generators/vue/templates/tsconfig.vitest.json.ejs

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "./tsconfig.app.json",
+  "exclude": [],
```

Comment:
we can delete this line?

Reply:
This exclude overrides exclude from the extended configuration file.

Reply:
ok